meta-ostro/conf/distro/ostro.conf

```diff
@@ -1,6 +1,12 @@
 DISTRO = "ostro"
 DISTRO_NAME = "Ostro OS"
-DISTRO_VERSION = "1.0~snapshot-${DATE}"
+# The version is bumped after each official release.
+# This implies that new releases go out with a version
+# string based on the predecessor plus the suffix, which
+# was deemed sufficient in the "[Ostro-dev] versioning Ostro OS"
```

Comment:
I find that to long and distracting. If someone is interested, they can find the discussion in their favorite location (local copy, via a web search engine, gmane if he had registered our list, etc.)  via the mail thread title.